flyteadmin/pkg/repositories/config/migrations.go

```diff
@@ -1182,7 +1182,43 @@ var NoopMigrations = []*gormigrate.Migration{
 	},
 }
 
-var Migrations = append(LegacyMigrations, NoopMigrations...)
+var ContinuedMigrations = []*gormigrate.Migration{
```

Comment:
can you add a comment here explaining what ContinuedMigrations is and that we should now add new migrations here?